docs/sources/clients/promtail/configuration.md

```diff
@@ -1944,6 +1944,8 @@ The optional `limits_config` block configures global limits for this instance of
 
 Maximum log line byte size allowed without dropping. Example: 256kb, 2M. 0 to disable.
 [max_line_size: <int> | default = 0]
+# Truncate log lines when they exceed max_line_size.
```

Comment:
Would prefer to have exact help message we have it on the flag parsing.
`Whether to truncate lines that exceed max_line_size. No effect if max_line_size is disabled`